src/main/resources/assets/farmersdelight/lang/it_it.json

```diff
@@ -192,7 +213,7 @@
 
   "itemGroup.farmersdelight": "Farmer's Delight",
 
-  "farmersdelight.container.recipe_book.cookable": "Mostra ciò che si può cuocere",
+  "farmersdelight.container.recipe_book.cookable": "Mostra cio' che si puo' cuocere",

```

Comment:
Quick question: why are instances of reverse-accents being converted to a loose hyphen?
Although I don't speak Italian, this seems like the wrong form of writing this accent. Just wanted to confirm.